control_panel_api/services.py

```diff
@@ -148,50 +133,39 @@ def delete_bucket_policies(bucket_name):
     aws.delete_policy(policy_arn_readonly)
 
 
-def detach_bucket_access_from_app_role(app_slug, bucket_name, access_level):
+def detach_bucket_access_from_app_role(
+        bucket_name, readwrite, app_role_name):
     policy_arn = _policy_arn(
         bucket_name=bucket_name,
-        readwrite=access_level == READWRITE
+        readwrite=readwrite
     )
 
     aws.detach_policy_from_role(
         policy_arn=policy_arn,
-        role_name=_app_role_name(app_slug)
+        role_name=app_role_name
     )
 
 
-def bucket_create(name):
-    create_bucket(name)
-    create_bucket_policies(name)
-
-
-def bucket_delete(name):
-    """Note we do not destroy the actual data, just the policies"""
-    delete_bucket_policies(name)
-
-
-def apps3bucket_create(apps3bucket):
+def attach_bucket_access_to_app_role(bucket_name, readwrite, app_role_name):
```

Comment:
Again, feels like these functions' names could be higher level and contains the words `grant`/`revoke` in them. 

But for now don't worry if it's too annoying to rename. The more I think about it, the more I like the idea of having `AWS*` "models" which performs these operations and have access to the models' instances.